drivers/pwm/pwm_nrf5_sw.c

```diff
@@ -12,6 +12,9 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(pwm_nrf5_sw);
 
+#define PWM_0_MAP_SIZE \
+	(DT_NORDIC_NRF_SW_PWM_SW_PWM_TIMER_CAPTURE_COMPARE_COUNT - 1)
```

Comment:
Why `- 1`?

Reply:
-1 because we need one compare register to set the period. So the last compare is used to "reset"

Reply:
Not needed anymore since we changed the DT name to be CHANNEL_COUNT - hence the MAP_SIZE is equal to channel count (user just have to make sure there the channel count is one less than the number of capture compare registers) - as mentioned in yaml description.